modules/creating-a-machine-pool-ocm.adoc

```diff
@@ -77,8 +77,8 @@ Alternatively, you can set your autoscaling preferences for the machine pool aft
 ====
 
 . If you did not enable autoscaling, select a compute node count:
-* If you deployed your cluster using a single availability zone, select a *Worker node count* from the drop-down menu. This defines the number of compute nodes to provision to the machine pool for the zone.
-* If you deployed your cluster using multiple availability zones, select a *Worker node count (per zone)* from the drop-down menu. This defines the number of compute nodes to provision to the machine pool per zone.
+* If you deployed your cluster using a single availability zone, select a *Compute node count* from the drop-down menu. This defines the number of compute nodes to provision to the machine pool for the zone.
```

Comment:
@laubai Hi, Could you please make similar changed for Line number 54 - `Select a *Worker node instance type* from the drop-down menu. The instance type defines the vCPU and memory allocation for each compute node in the machine pool`.? 
There is no "Worker node instance type" in OCM UI. Instead it needs to be "Compute node instance type".

Reply:
Ah, thankyou, I missed that on my way through.